### PR TITLE
chore(docs): adds QuestDB to databases config page

### PR DESCRIPTION
**Pull request by goodroot**
### SUMMARY

Adds QuestDB to the database pages. We offer similar guidance in our documentation.

https://superset.apache.org/docs/configuration/databases

https://questdb.io/docs/third-party-tools/superset/

### CODE REVIEW

File: docs/docs/configuration/databases.mdx [hunk 1]
```diff
@@ -1151,6 +1152,40 @@ SSL Secure extra add json config to extra connection information.
 }
 ```
 
+#### QuestDB
+
+[QuestDB](https://questdb.io/) is a high-performance 
+[time-series database](https://questdb.io/glossary/time-series-database/) that 
+specializes in hardware effiency and high-cardinality data. It's ideal for
```
Comment (rusackas) [inline docs/docs/configuration/databases.mdx:1159]:
```suggestion
specializes in hardware efficiency and high-cardinality data. It's ideal for 
```

**Review by rusackas — APPROVED** [review]
Caught a typo, but otherwise this looks great to me! Thank you! Feel free to add the logo to the website/docs, too, if you'd like. 

@betodealmeida may also be able to help make sure QuestDB is covered here on the compatibility matrix:
https://github.com/apache/superset/tree/master/superset/db_engine_specs

**Comment by goodroot** [comment]
@rusackas Yes! Thank you. Heh, vacation next week. Goo-brain for sure.

Added the logo, appreciate the invitation.

Happy to help with matrix if I can?

**Comment by eschutho** [comment]
Oh we also have this [db engine spec](https://github.com/apache/superset/pull/24172) PR  for QuestDB.

**Comment by goodroot** [comment]
Miguel, the author of that PR, is no longer with QuestDB unfortunately.

It may require Superset-side assistance.

**Pull request closed without merging**